hack/ibabou/yamlSerializer.go

```diff
@@ -0,0 +1,30 @@
+package ibabou
```

Comment:
This is great package name 👍

Reply:
Everything generally looks good. Lots of version numbers changing, hoping no typos on that front. Thanks Ibrahim!